membership-attribute-service/conf/riff-raff.yaml

```diff
@@ -10,7 +10,7 @@ deployments:
     parameters:
       templatePath: membership-attribute-service.yaml
       amiTags:
-        Recipe: jammy-membership-java8
+        Recipe: jammy-membership-java11
```

Comment:
update to java 11 because play Jars are built against 11, so I got an error about class file versions.